cmd/dm-ctl/main.go

```diff
@@ -31,37 +31,98 @@ import (
 	"github.com/pingcap/dm/pkg/utils"
 )
 
+func helpUsage(cfg *common.Config) {
+	fmt.Println("Usage: dmctl [global options] command [command options] [arguments...]")
+	fmt.Println()
+	ctl.Usage()
+	fmt.Println()
+	fmt.Println("Special Command:")
+	f := cfg.FlagSet.Lookup(common.EncryptCmdName)
+	fmt.Println(fmt.Sprintf("\t--%s %s", f.Name, f.Usage))
+	fmt.Println()
+	fmt.Println("Global Options:")
+	cfg.FlagSet.VisitAll(func(flag2 *flag.Flag) {
+		if flag2.Name == common.EncryptCmdName {
+			return
+		}
+		fmt.Println(fmt.Sprintf("\t--%s %s", flag2.Name, flag2.Usage))
+	})
+}
+
 func main() {
 	cfg := common.NewConfig()
-	err := cfg.Parse(os.Args[1:])
-	switch errors.Cause(err) {
-	case nil:
-	case flag.ErrHelp:
-		os.Exit(0)
-	default:
-		fmt.Printf("parse cmd flags err: %s", err)
-		os.Exit(2)
+	args := os.Args[1:]
```

Comment:
can you add some test cases with different arguments (and formats) for this binary?

Reply:
sure, how about integration_test?

Reply:
OK

Reply:
add integration test.
check command with
```1. ./dmctl
2. ./dmctl --help
3. ./dmctl start-task
4. ./dmctl start-task --help
5. ./dmctl --master-addr xxx start-task task.yaml
6. ./dmctl --master-addr query-status
```

1, 2  should return the same help message (global message)
3, 4 should return the same help message (subCommand message)
5 and 6 just as same as other integration test